examples/social-share.amp.html

```diff
@@ -43,6 +36,8 @@
 
     <div class="social-box">
       <amp-social-share type="email" data-param-subject="Hello World" data-param-body="What's up?"></amp-social-share>
+      <amp-social-share type="whatsapp"></amp-social-share>
+      <amp-social-share type="pinterest"></amp-social-share>
```

Comment:
Shall we add a `data-param-media` for pinterest example? For example an amp logo image. Maybe also in amp-by-example?